src/tests/constants.py

```diff
@@ -1,3 +1,5 @@
+from .auth.auth import BearerAuth
```

Comment:
not neccassary since py3 but wont harm if someone uses other editors: 
add:  # coding: utf-8